shared/src/commonMain/kotlin/org/hyperskill/app/notification/remote/domain/model/PushNotificationData.kt

```diff
@@ -5,6 +5,8 @@ import kotlinx.serialization.Serializable
 
 @Serializable
 data class PushNotificationData(
+    @SerialName("notification_id")
+    val backendId: String? = null,
```

Comment:
@XanderZhu can't use `id` here, because of a conflict with the Android extension
https://github.com/hyperskill/mobile-app/blob/f58e2b83a731c12dd84524305d8d61395684c962/androidHyperskillApp/src/main/java/org/hyperskill/app/android/notification/remote/model/PushNotificationDataExtensions.kt#L8

Also, this is optional until the backend deploys this to the production